utils/dynamic-importer.js

```diff
@@ -83,3 +83,15 @@ export function loadTranslation(name) {
   // Note: directly returns the import, not a function
   return import(/* webpackChunkName: "[request]" */ `@/assets/translations/${name}.yaml`);
 }
+
+export function importComponent(path) {
```

Comment:
This is the same as `importEdit` from above

Reply:
Done